tests/testthat/test_cran_metrics.R

```diff
@@ -0,0 +1,21 @@
+context("Test CRAN related metrics")
+
+test_that("CRAN current version returns correct format", {
+
+    expect_equal(
+      grep(
+        "[0-9]+.[0-9]+.[0-9]+(.[0-9]+)*",
```

Comment:
Be careful with the `.` usage here - `.` is a reserved character in `regex` and will capture any characters such that "1a2b3cd" would be a valid version number. Not a huge deal, and the standard version format will still get picked up. 

Consider using `"\\d+(?:\\.\\d+){2,}"`

Reply:
oh dear, regular expressions certainly are not my strong suit ;) thx for pointing that out!